GNUmakefile

```diff
@@ -90,15 +90,17 @@ generate: abigen codecgen mockery ## Execute all go:generate commands.
 	go generate -x ./...
 
 .PHONY: testscripts
-testscripts: chainlink-test ## Install and run testscript against testdata/scripts/* files.
+testscripts: install-chainlink ## Install and run testscript against testdata/scripts/* files.
```

Comment:
why was a change to the makefile needed? add something the PR commit message for context the tangential fix made here

Reply:
Yea, I needed to change it because it doesn't work. `chainlink-test` runs `go build`, which means the chainlink binary has to be referenced by path (i.e. `./chainlink`). `go install` makes the binary available on $PATH, so it can be referenced without a path (i.e. `chainlink`)

Reply:
Oh I am remembering now that this is by design. It is preferable not to affect the locally installed version in order to run a test. The previous behavior isolated the effects to the dev environment. Let's revert this.

Reply:
That seems like a development workflow footgun to me. I'll push this change to a different PR so we don't block this one and we can discuss more